packages/js/customer-effort-score/changelog/enhancement-38960

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+New product block editor modal text enhancements
```

Comment:
We should change this changelog to be CES specific.
In this case we are fixing the onsubmit label.